src/distance.cpp

```diff
@@ -0,0 +1,64 @@
+#include <math.h>
```

Comment:
Make any lines longer than 80 characters multiple lines! Check out cpplint.py to help you do this (extension in VS Code) if you want an easy way to do this

Reply:
Is this what you meant?

`Vec3 SphericalDistanceDeterminationAlgorithm::getCenter(Vec3* spats) {
    Vec3 diff1 = std::move(spats[1] - spats[0]);
    Vec3 diff2 = std::move(spats[2] - spats[1]);

    Vec3 circleN = std::move(diff1.CrossProduct(diff2));
    Vec3 circlePt = spats[0];

    Vec3 mid1 = std::move(midpoint(spats[0], spats[1]));
    Vec3 mid2 = std::move(midpoint(spats[1], spats[2]));

    Vec3 mid1N = std::move(diff1);
    Vec3 mid2N = std::move(diff2);

    Mat3 matrix;
    matrix = {circleN.x, circleN.y, circleN.z, mid1N.x, mid1N.y,
              mid1N.z, mid2N.x, mid2N.y, mid2N.z};

    int alpha = circleN*circlePt;
    int beta = mid1N*mid1;
    int gamma = mid2N*mid2;

    Vec3 y = {alpha, beta, gamma};

    Vec3 center = std::move(matrix.Inverse() * y);

    return center;
}`